### DISCUSSION TITLE
Spanish Translation

### DISCUSSION THREAD

**Discussion by delfino-nunez** [borgbase/vorta#1225] [Translations]
Hi,
I have finished the Spanish translations in Transifex and they are ready to go.
Do you build the ts files from there and add them to the project ?
Or should I create a PR to add them?

Thank you

![image](https://user-images.githubusercontent.com/770967/159112395-a05d218f-4223-4f78-87bd-88868993fe42.png)

**Comment by m3nu** [comment]
Translations are pulled in from the Transifex API. Will be included in the next release.

Thanks for the great work to translate!

**Reply by delfino-nunez** [reply]
Awesome, thank you.

**Comment by adrianlopezgalera** [comment]
> Awesome, thank you.

Hi! Hello. I haven't had the opportunity to use your translation because I don't see a way to change the language of the program's interface without changing the system language.

In any case, keep in mind that "reactivación" is written together in Spanish. Please be careful not to copy English spelling.

Regards.
